ray-operator/controllers/common/pod.go

```diff
@@ -260,7 +260,31 @@ func setMissingRayStartParams(rayStartParams map[string]string, nodeType rayiov1
 }
 
 // concatenateContainerCommand with ray start
-func concatenateContainerCommand(nodeType rayiov1alpha1.RayNodeType, rayStartParams map[string]string) (fullCmd string) {
+func concatenateContainerCommand(nodeType rayiov1alpha1.RayNodeType, rayStartParams map[string]string, resource v1.ResourceRequirements) (fullCmd string) {
+
+	if _, ok := rayStartParams["num-cpus"]; !ok {
+		cpu := resource.Limits[v1.ResourceCPU]
+		if !cpu.IsZero() {
+			rayStartParams["num-cpus"] = strconv.FormatInt(cpu.Value(), 10)
+		}
+	}
+
+	if _, ok := rayStartParams["object-store-memory"]; !ok {
```

Comment:
`object-store-memory` is not equals to container memory.  Maybe `--memory` will be a matching option.

Reply:
fix it